app/portainer/models/dockerhub.js

```diff
@@ -1,6 +1,6 @@
 export function DockerHubViewModel(data) {
   this.Name = 'DockerHub';
-  this.URL = '';
+  this.URL = data.URL;
```

Comment:
Default the name to `DockerHub (anonymous)` and URL to `docker.io`
Remove Authentication related stuff (Auth / Username / Password).
Remove `data` parameter